docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1236,6 +1236,15 @@ As the core components use emotion as a styled engine, the props used by emotion
   +<Input size="small" />
   ```
 
+- Set the InputAdornment `position` prop to `start` or `end`. Use `start` if used as the value of the `startAdornment` prop. Use `end` if used as the value of the `endAdornment` prop.
+
+  ```diff
+  -<TextField endAdornment={<InputAdornment>Kg</InputAdornment>} />
+  +<TextField endAdornment={<InputAdornment position="end">Kg</InputAdornment>} />
```

Comment:
A nip, git diff groups the removals and the additions. I think that

```
-
-
+
+
```

Would be clearer.